src/emoji-row.jsx

```diff
@@ -25,14 +25,8 @@ export default class EmojiRow extends Component {
               ariaLabel={emoji.name}
               role="option"
               key={emoji.unicode}
-              onClick={function(e) {
-                onChange(emoji);
-              }}
-              onKeyUp={function(e) {
-                e.preventDefault();
-                if (e.which === 13 || e.which === 32) {
-                  onChange(emoji);
-                }
+              onSelect={() => {
```

Comment:
The Emoji component is also used in the category list in the header, where the parent component needs the category id instead of the emoji object when the Emoji component is selected. I decided to keep the contract as is since it doesn't assume anything about what the parent component will do when the Emoji is selected. Performance is still good since updates are prevented at the row level.